examples/window-app/efr32/src/lcd.c

```diff
@@ -0,0 +1,188 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <AppConfig.h>
+#include <display.h>
+#include <dmd.h>
+#include <glib.h>
+#include <lcd_cover.h>
+
+#define COVERING_TYPE_ROLLERSHADE 0                  // Lift
+#define COVERING_TYPE_ROLLERSHADE_2_MOTOR 1          // Lift
+#define COVERING_TYPE_ROLLERSHADE_EXTERIOR 2         // Lift
+#define COVERING_TYPE_ROLLERSHADE_EXTERIOR_2_MOTOR 3 // Lift
+#define COVERING_TYPE_DRAPERY 4                      // Lift
+#define COVERING_TYPE_AWNING 5                       // Lift
+#define COVERING_TYPE_SHUTTER 6                      // Tilt
+#define COVERING_TYPE_TILT_BLIND 7                   // Tilt
+#define COVERING_TYPE_TILT_LIFT_BLIND 8              // Lift, Tilt
+#define COVERING_TYPE_PROJECTOR_SCREEN 9             // Tilt
+
+#define BAND_COUNT 12
+#define BAND_SIZE (LCD_COVER_SIZE / BAND_COUNT)
+
+extern GLIB_Context_t glibContext;
```

Comment:
nit: Please move declaration to a header file that is included by the file that defines this variable.